src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -2445,9 +2379,6 @@ void GetGroundHeatExchangerInput(EnergyPlusData &state)
     //       AUTHOR:          Dan Fisher
     //       DATE WRITTEN:    August, 2000
     //       MODIFIED         Arun Murugappan
-    //       RE-ENGINEERED    na
-
-    bool errorsFound = false;
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:2450]:(style),[variableScope],The scope of the variable 'errorsFound' can be reduced.